crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -389,6 +389,14 @@ def prepare_nodes_for_os_upgrade
     end
 
     commit_and_check_proposal
+
+    # FIXME: upgrade of the nodes needs to be correctly orchestrated, to upgrade nodes one by one
+    # Now we're adding this only only to imitate original autoyast-based upgrad and to
+    # have current upgrade CI job working
+    upgrade_nodes.each do |node|
+      Rails.logger.info("initiating upgrade of node #{node.name}")
+      node.ssh_cmd("/usr/sbin/crowbar-upgrade-os.sh")
```

Comment:
No, the script is already there, on all nodes, created by the proposal commit at line 391 (see https://github.com/crowbar/crowbar-core/pull/640 for details). 
But now it is not executed. 
By imitation, I mean: when we execute the script on every node at once, we do not care about non-disruptive upgrade, we just upgrade all nodes to SP2, like we did with the old upgrade path.